Jenkinsfile

```diff
@@ -1,5 +1,7 @@
 buildPlugin(
-    useAci: true,
-    failFast: false,
-    configurations: buildPlugin.recommendedConfigurations()
-)
+  useContainerAgent: true,
+  failFast: false,  
+  configurations: [
+    [platform: 'linux', jdk: 8],
```

Comment:
I believe previous configuration builds against jdk 11 or 17 while this one only against 8

Reply:
Nope, as we can see on the master branch, it was all Java 8: https://github.com/jenkinsci/build-history-manager-plugin/runs/12000577420